SCALESET_README.md

```diff
@@ -0,0 +1,44 @@
+## Run init.sh
+
+If you're using Codespaces or VS Code Remote Devcontainers, this script will automatically run for you. Otherwise, you can manually run `var/init.sh/` to (re)create a kind cluster and install a certificate manager in it.
+
+## Export GITHUB_TOKEN
+
+Create a GitHub PAT with the [appropriate permissions](https://github.com/thboop/actions-runner-controller#deploying-using-github-app-authentication)
+
+Make sure the PAT has the right permissions depending on your use case (repo only, administrator of organisation, or enterprise)
+
+`export GITHUB_TOKEN=ghp_***************************`
+
+## Build new ARC image
+
+Set `$DOCKER_USER` to your docker repository user (export DOCKER_USER={docker.io-uname})
+
+Run 
+
+`docker login`
+
+so that you can push the image you're building below with
+
+`make docker-buildx`
+
+## Make Release
+
+Run 
+
+`make release`
+
+in your project root directory.
+
+## Deploy the k8s manifest 
+
+```
+kubectl create -f release/actions-runner-controller.yaml \
+&& kubectl create secret generic controller-manager \
+    -n actions-runner-system \
+    --from-literal=github_token=${GITHUB_TOKEN}
+```
+
+From here on, we will NOT be following the [ARC docs](https://github.com/actions-runner-controller/actions-runner-controller#usage) to manage our runners.
+
+## Create an autoscaler client
```

Comment:
Is this coming later?

Reply:
Yes - I'll actually remove SCALESET_README.md to avoid confusion 👍